runtime/coverage.go

```diff
@@ -483,3 +484,45 @@ func (r *CoverageReport) UnmarshalJSON(data []byte) error {
 
 	return nil
 }
+
+// MarshalLCOV serializes each common.Location/*LocationCoverage
+// key/value pair on the *CoverageReport.Coverage map, to the
+// LCOV format. Currently supports only line coverage, function
+// and branch coverage are not yet available.
```

Comment:
Could you please add a link to a description of the file format here?

I found it in lcov's geninfo man page, https://github.com/linux-test-project/lcov/blob/master/man/geninfo.1#L948, not sure if there's a better description somewhere else.

Reply:
Looks quite official, so I am going with this :+1:
Added in https://github.com/onflow/cadence/pull/2577/commits/117d51eb2a1e623d9a61613a5a4b2be55a6b1e88